g2p_registry_rest_api/data/default_api_users.xml

```diff
@@ -0,0 +1,16 @@
+<!-- pylint:disable=create-user-wo-reset-password -->
+<odoo noupdate="1">
+
+    <record id="default_api_user" model="res.users">
+        <field name="name">Default API User</field>
+        <field name="login">defapiuser</field>
+        <field name="password">@xs!23HjU87P3@nUt5</field>
```

Comment:
Let's not create this user automatically here. We can do this in the demo, but not in the module itself as most people will forget to change it, becoming a vulnerability issue.

Reply:
I will remove this in the next push